src/main/java/edu/kis/powp/jobs2d/drivers/visitor/IVisitor.java

```diff
@@ -2,10 +2,14 @@
 
 import edu.kis.powp.jobs2d.Job2dDriver;
 import edu.kis.powp.jobs2d.drivers.DriverComposite;
+import edu.kis.powp.jobs2d.drivers.LineSimulatorDriver;
 import edu.kis.powp.jobs2d.drivers.adapter.DriverDecorator;
+import edu.kis.powp.jobs2d.drivers.logger.LoggerDriver;
 
 public interface IVisitor {
     void visit(DriverComposite driver);
     void visit(Job2dDriver driver);
```

Comment:
Instead of visit(Job2dDriver) introduce intermediary interface VisitableJob2dDriver. Visiting Job2dDriver via accept is not always possible.